docs/misc/mainnet_spoon.md

```diff
@@ -0,0 +1,124 @@
+chain).
+
+So we need to select a new list of validators to start off our
+chain. Suppose that we want our chain to have two validators,
+`validator0.near` and `validator1.near`. Let's make a new directory
+where we'll be storing intermediate files during this process:
+
+```shell
+$ mkdir ~/test-chain-scratch
+```
+
+then using your favorite editor, lay out the validators you want in
+the test chain as a JSON list in the same format as the `validators`
+field in `genesis.json`, maybe in the file
+`~/test-chain-scratch/validators.json`
+
+```json
+[
+  {
+    "account_id": "validator0.near",
+    "public_key": "ed25519:GRAFkrqEkJAbdbWUgc6fDnNpCTE83C3pzdJpjAHkMEhq",
+    "amount": "100000000000000000000000000000000"
+  },
+  {
+    "account_id": "validator1.near",
+    "public_key": "ed25519:5FxQQTC9mk5kLAhTF9ffDMTXiyYrDXyGYskgz46kHMdd",
+    "amount": "100000000000000000000000000000000"
+  }
+]
+```
+
+This is also a good time to think about what extra accounts you might
+want in your test chain. Since all accounts in the test chain will
+have the same keys as they do on mainnet, you'll only have access to
+the accounts that you have access to on mainnet. If you want to add an
+account with a large balance to properly test things out, you can
+write them out in a file as a JSON list of state records (in the same
+format as they appear in `records.json`). For example, you could put
+the following in `~/test-chain-scratch/extra-records.json`:
+
+```json
+[
+  {
+    "Account": {
+      "account_id": "my-test-account.near",
+      "account": {
+        "amount": "10000000000000000000000000000000000",
+        "locked": "0",
+        "code_hash": "11111111111111111111111111111111",
+        "storage_usage": 182,
+        "version": "V1"
+      }
+    }
+  },
+  {
+    "AccessKey": {
+      "account_id": "my-test-account.near",
+      "public_key": "ed25519:Eo9W44tRMwcYcoua11yM7Xfr1DjgR4EWQFM3RU27MEX8",
+      "access_key": {
+        "nonce": 0,
+        "permission": "FullAccess"
+      }
+    }
+  }
+]
+```
+
+You'll want to include an access key here, otherwise you won't be able
+to do anything with the account. Note that here you can also add
+access keys for any mainnet account you want, so you'll be able to
+control it in the test chain.
+
+Now to make these changes to the genesis and records files, you can
+use the `neard amend-genesis` command like so:
+
+```shell
+# mkdir ~/near-test-chain/
+$ neard amend-genesis --genesis-file-in $NEAR_HOME_DIRECTORY/output/genesis.json --records-file-in $NEAR_HOME_DIRECTORY/output/records.json --validators ~/test-chain-scratch/validators.json --extra-records ~/test-chain-scratch/extra-records.json --chain-id $TEST_CHAIN_ID --records-file-out ~/near-test-chain/records.json --genesis-file-out ~/near-test-chain/genesis.json
+```
+
+Then you can use the resulting genesis and records files in
+`~/near-test-chain/` to start your new chain as you normally would.
```

Comment:
What “normally would” is was still non-trivial to me to figure out. Since we have held reader’s hand until this point, lets see this through to the end. Can this mention that one ought to set corresponding fields in `config.json`, and give an example of what that looks like with the modifications applied?

Finally, worthwhile to mention that the start-up will take a long while for the first time, depending on the size of the state.

Reply:
added some more